pilot/pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -110,14 +111,15 @@ spec:
         - "80"
         - -d
         - "15020"
-        image: docker.io/istio/proxy_init:unittest
+        image: gcr.io/istio-release/proxy_init:master-latest-daily
         imagePullPolicy: IfNotPresent
         name: istio-init
         resources: {}
         securityContext:
           capabilities:
             add:
             - NET_ADMIN
+          privileged: true
```

Comment:
is `privileged: true` the desired behavior?  @costinm @andraxylia

Reply:
It is at least the default behavior. It is not set and the yaml.Marshal() writes default values as well. Please let me know if this is not desired. That would be strange.

Reply:
Do not worry about this for now: the removal of privilege is not yet cherry picked in master, I will have a separate PR.